packages/tag/test/mdx-tag.test.js

```diff
@@ -0,0 +1,31 @@
+import fs from 'fs'
+import React from 'react'
+import { renderToString } from 'react-dom/server'
+
+import { MDXTag } from '../src'
+
+const H1 = props => <h1 style={{ color: 'tomato' }} {...props} />
+
+it('Should render the desired component', async () => {
+  const result = renderToString(
+    <MDXTag name="h1" components={{ h1: H1 }} children="Hello, world!" />
+  )
+
+  expect(result).toMatch(/style="color:tomato"/)
+})
+
+it('Should render the layout component', async () => {
+  const Layout = ({children}) => <div id="layout">{children}</div>
+  const components = { h1: H1 }
+  const result = renderToString(
+    <MDXTag Layout={Layout} name="wrapper" components={components}>
+      <MDXTag Layout={Layout} name="h1" components={{ h1: H1 }} />     
```

Comment:
Not sure why this line also has a `Layout` prop.

Reply:
Because I copy-pasted the previous line and edited 🤣 Will update

Reply:
Fixed